org.getmonero.Monero.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: org.getmonero.Monero
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --device=dri
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: monero-wallet-gui
+modules:
+  - name: monero-bin
+    buildsystem: simple
+    sources: 
+    - type: archive
+      url: https://downloads.getmonero.org/gui/monero-gui-linux-x64-v0.17.1.9.tar.bz2
```

Comment:
Technically speaking, it would be nice to build it from source, but I imagine it's not necessarily the most important thing for a wallet.

Reply:
it would allow supporting aarch64 as well....

Reply:
FYI, currently you (try to) build an aarch64 flatpak with a x86_64 binary.